src/Core/Framework/Rule/RuleComparison.php

```diff
@@ -9,6 +9,9 @@
 #[Package('fundamentals@after-sales')]
 class RuleComparison
 {
+    public const FORMAT_DATE = 'Y-m-d';
```

Comment:
we already have date(time) formats in the Defaults https://github.com/shopware/shopware/blob/trunk/src/Core/Defaults.php#L33-L38

are they also applicable here?